### DISCUSSION TITLE
Edge is not selectable in a group node.

### DISCUSSION THREAD

**Discussion by Kost927** [xyflow/xyflow#3343] [Q&A]
Have a problem. When the Edge is inside a group nod it is not selectable. So I have to detach the node to select an edge. How can I fix this issue?

**Comment by moklick** [comment]
By default the edges are below the node. You can increase the zIndex of an edge by using the `zIndex` edge option to change that.

**Reply by Kost927** [reply]
Thank you so much! It works.
